applications/asset_tracker/src/cloud_codec/cloud_codec.c

```diff
@@ -195,10 +230,34 @@ static const char *const channel_type_str[] = {
 	[CLOUD_CHANNEL_DEVICE_INFO] = CLOUD_CHANNEL_STR_DEVICE_INFO,
 	[CLOUD_CHANNEL_LIGHT_SENSOR] = CLOUD_CHANNEL_STR_LIGHT_SENSOR,
 };
-
+static struct cloud_sensor_chan_cfg sensor_cfg[] = {
+	{ .chan = CLOUD_CHANNEL_TEMP,
+	  .cfg = { .value = { [SENSOR_CHAN_CFG_ITEM_TYPE_SEND_ENABLE] = true } } },
+	{ .chan = CLOUD_CHANNEL_HUMID,
+	  .cfg = { .value = { [SENSOR_CHAN_CFG_ITEM_TYPE_SEND_ENABLE] = true } } },
+	{ .chan = CLOUD_CHANNEL_AIR_PRESS,
+	  .cfg = { .value = { [SENSOR_CHAN_CFG_ITEM_TYPE_SEND_ENABLE] = true } } },
+	{ .chan = CLOUD_CHANNEL_AIR_QUAL,
+	  .cfg = { .value = { [SENSOR_CHAN_CFG_ITEM_TYPE_SEND_ENABLE] = true } } },
+	{ .chan = CLOUD_CHANNEL_LIGHT_RED,
+	  .cfg = { .value = { [SENSOR_CHAN_CFG_ITEM_TYPE_SEND_ENABLE] = true } } },
+	{ .chan = CLOUD_CHANNEL_LIGHT_GREEN,
+	  .cfg = { .value = { [SENSOR_CHAN_CFG_ITEM_TYPE_SEND_ENABLE] = true } } },
+	{ .chan = CLOUD_CHANNEL_LIGHT_BLUE,
+	  .cfg = { .value = { [SENSOR_CHAN_CFG_ITEM_TYPE_SEND_ENABLE] = true } } },
+	{ .chan = CLOUD_CHANNEL_LIGHT_IR,
+	  .cfg = { .value = { [SENSOR_CHAN_CFG_ITEM_TYPE_SEND_ENABLE] = true } } }
+};
 static cloud_cmd_cb_t cloud_command_cb;
 struct cloud_command cmd_parsed;
 
+static int cloud_cmd_handle_sensor_set_chan_cfg(struct cloud_command const
+		      	  	  	  	  	  	  	  	  	*const cmd);
+
+static int cloud_set_chan_cfg_item(const enum cloud_channel channel,
+		      	  	  	  	  	   const enum sensor_chan_cfg_item_type type,
+								   const double value);
```

Comment:
Still weird things going on with the formatting, but could be for me locally on github

Reply:
This actually looks off and should be fixed. The CI actually did not run over this PR yet, I hope that `checkpatch` will notice such issues.

Reply:
this is the format given by both clang-format and uncrustify...
for line 254 it gives:
```static int
cloud_cmd_handle_sensor_set_chan_cfg(struct cloud_command const *const cmd);```

Reply:
i have no preference on these matters.  i would like to find a tool that produces the desired output, to save everyone time.